src/cli/commands/init.js

```diff
@@ -89,12 +91,17 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       question: 'license',
       default: String(config.getOption('init-license')),
     },
+    {
+      key: 'private',
+      question: 'private',
+      default: '',
```

Comment:
also the default should probably be "false" anyway so that when the question is printed tot he user they can see that it is explicitly false by default. Empty string might make someone question if the default is actually going to end up true or false (in reality it depends how "yn" interprets empty string)